src/pages/pt-BR/guides/rss.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: RSS
+description: Uma introdução a RSS em Astro
+i18nReady: true
+---
+
+Astro suporta geração rápida e automática de feeds RSS para blogs e outros websites orientados a conteúdo. Para mais informações sobre feeds RSS como um todo, veja [aboutfeeds.com](https://aboutfeeds.com/).
+
+## Usando `@astrojs/rss` (recomendado)
+
+O pacote `@astrojs/rss` providencia utilitários para a geração de feeds RSS utilizando [endpoints de API](/pt-BR/core-concepts/astro-pages/#páginas-não-html). Isso permite construções estáticas *e* geração sob demanda quando você estiver utilizando um [adaptador de SSR](/pt-BR/guides/server-side-rendering/#habilitando-o-ssr-em-seu-projeto).
+
+Primeiro, instale `@astrojs/rss` utilizando seu gerenciador de pacotes favorito:
+
+```bash
+# npm
+npm i @astrojs/rss
+# yarn
+yarn add @astrojs/rss
+# pnpm
+pnpm i @astrojs/rss
+```
+
+Então, garanta de que você [configurou `site`](/pt-BR/reference/configuration-reference/#site) no `astro.config` do seu projeto. Você utilizará essa opção para gerar os links do seu feed RSS [a partir da variável de ambiente `SITE`](/pt-BR/guides/environment-variables/#default-environment-variables).
+
+> Nota: A variável de ambiente `SITE` apenas existe na recente versão Astro 1.0 beta. Atualize para a versão mais recente do Astro (`astro@latest`), ou escreva seu `site` manualmente se isso não for possível (veja exemplos abaixo).
+
+Agora, vamos gerar nosso primeiro feed RSS! Crie um arquivo `rss.xml.js` no seu diretório `src/pages/`. `rss.xml` vai ser a URL final, então sinta-se livre para renomeá-lo se preferir.
+
+Em seguida, importe o utilitário `rss` do pacote `@astrojs/rss` e o chame com os seguintes parâmetros:
+
+```js
+// src/pages/rss.xml.js
+import rss from '@astrojs/rss';
+
+export const get = () => rss({
+    // campo `<title>` no xml final
+    title: 'Blog do Buzz',
+    // campo `<description>` no xml final
+    description: 'O guia de um humilde Astronauta para as estrelas',
+    // URL base para links <item> do RSS
+    // SITE irá utilizar o "site" do astro.config do seu projeto.
+    site: import.meta.env.SITE,
+    // lista de `<item>`s no xml final
+    // exemplo simples: gerar itens para cada arquivo md em /src/pages
+    // veja a seção "Gerando Items" para saber o frontmatter necessário e casos de uso mais avançados
```

Comment:
That word (`frontmatter`) seems to not have a default style of writing front matter or frontmatter : (.

Reply:
Yeah :sweat_smile:. I guess we can default it to `frontmatter` for now? In the future I'll do a major review of all pt-BR pages and change the other pages using `front matter`.